time start_time
time stop_time
---
bool is_deleted
